rake_nltk/rake.py

```diff
@@ -184,18 +190,26 @@ def _generate_phrases(self, sentences):
     def _get_phrase_list_from_words(self, word_list):
         """Method to create contender phrases from the list of words that form
         a sentence by dropping stopwords and punctuations and grouping the left
-        words into phrases. Ex:
+        words into phrases. All the phrases should have a length in the range
+        [self.min_length, self.max_length]. Ex:
 
         Sentence: Red apples, are good in flavour.
         List of words: ['red', 'apples', ",", 'are', 'good', 'in', 'flavour']
         List after dropping punctuations and stopwords.
         List of words: ['red', 'apples', *, *, good, *, 'flavour']
         List of phrases: [('red', 'apples'), ('good',), ('flavour',)]
 
+        List of phrases with a correct length:
+        For the range [1, 2]: [('red', 'apples'), ('good',), ('flavour',)]
+        For the range [1, 1]: [('good',), ('flavour',)]
+        For the range [2, 2]: [('red', 'apples')]
+
         :param word_list: List of words which form a sentence when joined in
                           the same order.
         :return: List of contender phrases that are formed after dropping
                  stopwords and punctuations.
         """
         groups = groupby(word_list, lambda x: x not in self.to_ignore)
-        return [tuple(group[1]) for group in groups if group[0]]
+        phrases = [tuple(group[1]) for group in groups if group[0]]
+        return [phrase for phrase in phrases
```

Comment:
Maybe 

```python
return [filter(lambda item: self.min_length <= len(item) <= self.max_length, phrases)]
```

Reply:
Thank you for the advice, really good for python beginner like me. 

Since this line contains 94 chars, which is longer than 80 chars per line and not compatible with your code style, I split it into 2 lines:

```python
        return [filter(
            lambda item: self.min_length <= len(item) <= self.max_length, phrases)]
```

Unfortunately, the second line still contains 84 chars :sweat: , so I changed it again:

```python
        return [filter(
            lambda x: self.min_length <= len(x) <= self.max_length, phrases)]
```

Do you think this is ok? Is it correct to put lambda expression in another line? Looking forward to your answer :smiley: